integration/asserts.go

```diff
@@ -0,0 +1,76 @@
+// +build requires_docker
+
+package main
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/cortexproject/cortex/integration/e2ecortex"
+)
+
+type ServiceType int
+
+const (
+	Distributor = iota
+	Ingester
+	Querier
+	QueryFrontend
+	TableManager
+	AlertManager
+)
+
+var (
+	// Service-specific metrics prefixes which shouldn't be used by any other service.
+	serviceMetricsPrefixes = map[ServiceType][]string{
+		Distributor:   []string{},
+		Ingester:      []string{},
+		Querier:       []string{},
+		QueryFrontend: []string{"cortex_frontend", "cortex_query_frontend"},
+		TableManager:  []string{},
+		AlertManager:  []string{},
+	}
+
+	// Blacklisted metrics prefixes across any Cortex service.
+	blacklistedMetricsPrefixes = []string{
+		"cortex_alert_manager", // It should be "cortex_alertmanager"
```

Comment:
I'm a bit confused where this is coming from?

Reply:
There's no such metric prefix. I just wanna avoid we'll add it in the future. I think it's a trivial mistake to do.

For example in the past we did a similar mistake with the query frontend and now we have mixed prefixes `cortex_query_frontend` and `cortex_frontend`.

Makes sense to you? If it's just my paranoia I can remove this.

Reply:
Fine to leave it IMO